nomad/state/state_store.go

```diff
@@ -2126,26 +2126,26 @@ func (s *StateStore) CSIVolumes(ws memdb.WatchSet) (memdb.ResultIterator, error)
 func (s *StateStore) CSIVolumeByID(ws memdb.WatchSet, namespace, id string) (*structs.CSIVolume, error) {
 	txn := s.db.ReadTxn()
 
-	watchCh, obj, err := txn.FirstWatch("csi_volumes", "id_prefix", namespace, id)
+	obj, err := txn.First("csi_volumes", "id", namespace, id)
```

Comment:
why the change from `FirstWatch` to `First`? won't this break blocking queries against `CSIVolume.Get`:
https://github.com/hashicorp/nomad/blob/9138d374d62a222159135a0b06ff91e3b134ea57/nomad/csi_endpoint.go#L208

Reply:
Good catch.

Reply:
Fixed in 5836f3b